src/petals/client/remote_sequential.py

```diff
@@ -28,6 +28,7 @@ def __init__(
         dht: Optional[DHT] = None,
         start_block: Optional[int] = None,
         end_block: Optional[int] = None,
+        **kwargs,
```

Comment:
this is a hook to pass things like adapter ID;

nb: the **kwargs are not passed infinitely; any args that are not explicitly declared in RemoteSequenceManager.__init__ will cause an error.